src/actions/eval-frame-action-validator.js

```diff
@@ -92,11 +92,25 @@ const schemas = {
   UPDATE_VALUE_IN_HISTORY: {
     type: "object",
     additionalProperties: false,
+    required: ["type", "historyItem"],
     properties: {
-      type: { type: "string" },
-      historyId: { type: "string" }
-    },
-    required: ["type", "historyId"]
+      type: {
+        type: "string"
+      },
+      historyItem: {
+        type: "object",
+        properties: {
+          historyId: { type: "string" },
+          content: { type: "string" },
+          language: { type: "string" },
+          level: { type: "string" },
+          historyType: { type: "string" },
+          lastRan: { type: "number" }
+        },
```

Comment:
do we need all of these element to be sent just to update a value? language, lastRan will never need updating, but i suspect we probably don't need a few more of these

Reply:
I suppose not